src/dom.ts

```diff
@@ -1,6 +1,8 @@
-import { document } from "./consts";
+import { document as _document } from "./consts";
 import { IObject, IEventMap } from "./types";
 
+const document = _document as Document;
```

Comment:
@xiaoxiangmoe 

Is it meaningful?

It would be better to set the type of doc in const.ts to Document.